src/02-crm/companies.apib

```diff
@@ -0,0 +1,179 @@
+## Companies [/companies]
+
+### companies.list [GET /companies.list]
+
+Get a list of companies.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + added_at
+                + order (enum[string], optional)
+                    + Members
+                        + desc
+            + Default
+                + (object)
+                    + field: `added_at`
+                    + order: `desc`
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `96a38bbf-24ed-4083-8a5c-20db92aa471e` (string)
+                + name: `Pied Piper` (string)
+                + business_type: `NV` (string)
+                + vat_number: `BE0899623035` (string)
+                + local_business_number: `63326426` (string)
+                + email: `info@piedpiper.eu` (string)
+                + invoice_email: `invoicing@example.com` (string, optional)
+                + telephones (array[Telephone])
+                + website: `https://piedpiper.com` (string)
+                + avatar (string)
+                + address (Address)
+                + iban: `BE12123412341234` (string)
+                + bic: `BICBANK` (string)
+                + language: `nl` (string)
+                + responsible_user (object)
+                    + type: `user` (string)
+                    + id: `9b99d8f8-183b-4c7b-8354-9b98b47a192e` (string)
+                + added_at: `2016-02-04T16:44:33+00:00` (string)
+                + updated_at: `2016-02-05T16:44:33+00:00` (string)
+
+### companies.info [GET /companies.info]
+
+Get details for a single company.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `e8d31ae7-8258-4fcd-9b2d-78f41b0aa5d5 (string)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `c5685b0f-87b8-456f-8d9e-a5866814c47a` (string)
+            + name: `Pied Piper` (string)
+            + business_type: `NV` (string)
+            + vat_number: `BE0899623035` (string)
+            + local_business_number: `63326426` (string)
+            + email: `info@piedpiper.eu` (string)
+            + invoice_email: `invoicing@example.com` (string, optional)
+            + telephones (array[Telephone])
+            + website: `https://piedpiper.com` (string)
+            + avatar (string)
+            + address (Address)
+            + delivery_address (Addressee)
```

Comment:
Missing

Reply:
They are there now, but are we changing this to an array before releasing? or not?

Reply:
I would prefer to release with an array.